src/components/General/NotesSection.vue

```diff
@@ -93,42 +93,42 @@ export default {
       if (this.order_by_date) {
         if (this.active_arrow_down_note) {
           if (this.currentView === 'TargetDetail') {
-            return this.getTargetNotes(parseInt(this.$route.params.id)).sort(
+            return this.getTargetNotes(this.$route.params.id).sort(
```

Comment:
we need to refactor this method to many if else, I can not understand what we are doing there